### ISSUE TITLE
Add endpoint for owned mosaics

### ISSUE THREAD

**Issue by evias** [nemtech/catapult-rest#11]
Similar to how it was possible to *get namespaces from an account*, there should be an endpoint to retrieve **mosaics owned per account**.

Reference: https://nemtech.github.io/api/endpoints.html#operation/getNamespacesFromAccount

**Comment by martinayora** [comment]
I was implementing this feature, when I realized this is already done. Would it suffice with the `/account` endpoint?

It retrieves among others the mosaic ids and amount of them for a given account Id.

**Comment by evias** [comment]
I agree here, this is already available in the getAccountInfo endpoint.

Do we foresee adding an endpoint for `mosaicsCreated` in accounts ? because I believe that is what this task was referring to.

**Comment by Vektrat** [comment]
this was confirmed as not so much relevant / needed, always open for implementing in case of need

**Comment by Vektrat** [comment]
Reopening, @dgarcia360 thinks this is important

> In my opinion, this endpoint is needed.
> Mosaic created != balance owned.
> You can be the mosaic creator, distribute all the mosaic units, and you won't be able to know that you created this mosaic.
> 
> Imagine that this mosaic is configured as "supply mutable", and you want to create more mosaic units from the user interface.
> 
> It will be very difficult to fetch the mosaic id, even impossible without this endpoint if you didn't saved the mosaicId somewhere.

**Comment by Vektrat** [comment]
@dgarcia360 please suggest endpoint design

i assume we will have a specific endpoint such as below, since adding extra queries to the `/account/{accountId}`(info) endpoint seems to add too much (most likely unnecessary) weight to the generic endpoint

`/account/{accountId}/mosaicsCreated`

REST dev note: to implement this we will  probably need to filter by owner{Address/PublicKey} in the `mosaics` collection - results will be returned in an `ownedMosaics`/`createdMosaics`/`mosaics` key of an object as a list of mosaic ids

**Comment by dgarcia360** [comment]
@Vektrat For some reason, I missed this mention.

I want something similar to ``/account/{accountId}/namespaces``, so here it is a proposal:

https://github.com/nemtech/nem2-openapi/pull/32/files

**Comment by Vektrat** [comment]
waiting for https://github.com/nemtech/nem2-openapi/pull/32#issuecomment-544459545
